src/pcapi/routes/adage_iframe/bookings.py

```diff
@@ -30,12 +32,26 @@ def book_educational_offer(authenticated_email: str, body: BookEducationalOfferR
         booking = educational_api.book_educational_offer(
             redactor_email=(body.redactorEmail), uai_code=body.UAICode, stock_id=body.stockId
         )
+    except AdageException:
+        logger.info("Could not book offer: adage api call failed", extra={"redactor_email": body.redactorEmail})
+        raise ApiErrors(
+            {"global": "L'appel api pour retrouver les informations du rédacteur a échoué"},
```

Comment:
```suggestion
            {"global": "La récupération des informations du rédacteur a échoué"},
```
Je me dis qu'ils n'ont pas besoin de savoir qu'on a fait un appel API

Reply:
```suggestion
            {"global": "La récupération des informations via Adage du rédacteur a échoué"},
```
? En fait je voulais mettre en évidence la partie qui pose problème (DB chez nous vs DB Adage :) )

Reply:
Ok pour moi